# HG changeset patch
# User Rik <rik@octave.org>
# Date 1382227370 25200
#      Sat Oct 19 17:02:50 2013 -0700
# Node ID 6516fc672fbb07a6866373ca2d0e7d0940791d02
# Parent  10d3bd4ec98148b6f14ec7cddddc8af3d820c721
maint: Re-categorize some functions between general/ and miscellaneous/ dirs.

* scripts/general/bincoeff.m: Moved from miscellaneous directory.

* scripts/miscellaneous/colon.m, scripts/miscellaneous/genvarname.m:
Moved from general directory.

* scripts/general/module.mk, scripts/miscellaneous/module.mk: Update build
system.

diff --git a/scripts/miscellaneous/bincoeff.m b/scripts/general/bincoeff.m
rename from scripts/miscellaneous/bincoeff.m
rename to scripts/general/bincoeff.m
diff --git a/scripts/general/module.mk b/scripts/general/module.mk
--- a/scripts/general/module.mk
+++ b/scripts/general/module.mk
@@ -3,41 +3,40 @@ FCN_FILE_DIRS += general
 general_PRIVATE_FCN_FILES = \
   general/private/__isequal__.m \
   general/private/__splinen__.m
 
 general_FCN_FILES = \
   general/accumarray.m \
   general/accumdim.m \
   general/bicubic.m \
+  general/bincoeff.m \
   general/bitcmp.m \
   general/bitget.m \
   general/bitset.m \
   general/blkdiag.m \
   general/cart2pol.m \
   general/cart2sph.m \
   general/cell2mat.m \
   general/celldisp.m \
   general/chop.m \
   general/circshift.m \
-  general/colon.m \
   general/common_size.m \
   general/cplxpair.m \
   general/cumtrapz.m \
   general/curl.m \
   general/dblquad.m \
   general/deal.m \
   general/del2.m \
   general/display.m \
   general/divergence.m \
   general/fieldnames.m \
   general/flipdim.m \
   general/fliplr.m \
   general/flipud.m \
-  general/genvarname.m \
   general/gradient.m \
   general/idivide.m \
   general/int2str.m \
   general/interp1.m \
   general/interp2.m \
   general/interp3.m \
   general/interpn.m \
   general/interpft.m \
diff --git a/scripts/general/colon.m b/scripts/miscellaneous/colon.m
rename from scripts/general/colon.m
rename to scripts/miscellaneous/colon.m
diff --git a/scripts/general/genvarname.m b/scripts/miscellaneous/genvarname.m
rename from scripts/general/genvarname.m
rename to scripts/miscellaneous/genvarname.m
diff --git a/scripts/miscellaneous/module.mk b/scripts/miscellaneous/module.mk
--- a/scripts/miscellaneous/module.mk
+++ b/scripts/miscellaneous/module.mk
@@ -1,21 +1,21 @@
 FCN_FILE_DIRS += miscellaneous
 
 miscellaneous_PRIVATE_FCN_FILES = \
   miscellaneous/private/display_info_file.m \
   miscellaneous/private/__xzip__.m
 
 miscellaneous_FCN_FILES = \
   miscellaneous/ans.m \
-  miscellaneous/bincoeff.m \
   miscellaneous/bug_report.m \
   miscellaneous/bunzip2.m \
   miscellaneous/bzip2.m \
   miscellaneous/cast.m \
+  miscellaneous/colon.m \
   miscellaneous/citation.m \
   miscellaneous/comma.m \
   miscellaneous/compare_versions.m \
   miscellaneous/computer.m \
   miscellaneous/copyfile.m \
   miscellaneous/debug.m \
   miscellaneous/delete.m \
   miscellaneous/desktop.m \
@@ -23,16 +23,17 @@ miscellaneous_FCN_FILES = \
   miscellaneous/dos.m \
   miscellaneous/dump_prefs.m \
   miscellaneous/edit.m \
   miscellaneous/error_ids.m \
 	miscellaneous/fact.m \
   miscellaneous/fileattrib.m \
   miscellaneous/fileparts.m \
   miscellaneous/fullfile.m \
+  miscellaneous/genvarname.m \
   miscellaneous/getappdata.m \
   miscellaneous/getfield.m \
   miscellaneous/gunzip.m \
   miscellaneous/gzip.m \
   miscellaneous/info.m \
   miscellaneous/inputname.m \
   miscellaneous/isappdata.m \
   miscellaneous/isdeployed.m \
